src/components/submission/show.vue

```diff
@@ -91,6 +89,7 @@ export default {
   setup() {
     const { project, resourceStates } = useRequestData();
     const { request, awaitingResponse } = useRequest();
+    const { form } = useForm();
```

Comment:
don't need the extra resources of useForm

```suggestion
    const { project, form, resourceStates } = useRequestData();
    const { request, awaitingResponse } = useRequest();
```